recipes/pyre/pyre_version.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 4a24372e..be57f32d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -33,8 +33,14 @@ include(pyre_cuda)
+ include(pyre_gsl)
+ include(pyre_mpi)
+ 
+-# ask git for the pyre version
+-pyre_getVersion()
++# set pyre version info
++set(REPO_MAJOR 1)
```

Comment:
Can you modify this to use the `$PKG_VERSION` environment variable so it doesn't get out of sync?